Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7613,6 +9640,31 @@ def main():
         elif demisto.command() == 'panorama-install-file-content-update':
             panorama_install_file_content_update_command(args)
 
+        elif demisto.command() == 'pan-os-platform-get-arp-tables':
+            topology = get_topology()
```

Comment:
why do we need to get topology on each one of the commands? use it before running all the if statements

Reply:
We don't need the setup costs of the topology object when a non-topology (legacy) command is being called. this saves a few API calls and makes the code more efficient.

Reply:
then i suggest to end the if's of the panorama commands and then start a new series of if statement with pan-os commands while you call this line ```topology = get_topology()```

Reply:
Can you think of a nice way to do this? If we  do it this way we will always create a topology object unless we add return to every other elif blocks. This is why I did it with the decorators, by the way, originally!

Reply:
But this will cause get_topology() to be evaluated every time, even if the command above it already matched. For example if the command is pan-os-install-file-content-update, then this command will run then topology will be created and no further action will be taken

Reply:
you are right, so just use it inside the if statement, i guess if would give us better grasp of what command is based on topology object.